### ISSUE TITLE
DIG app amd myDIG don't work on Firefox or Safari

### ISSUE THREAD

**Issue by szeke** [usc-isi-i2/dig-etl-engine#63]
The browser shows a blank screen after login in myDIG

**Comment by szeke** [comment]
Thomas fixed the DIG app. For myDIG, this issue depends on #25

**Comment by szeke** [comment]
Closing, won't fix for the current version, waiting for the new GUI
